sources/packages/api/src/database/entities/application-status.type.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Possible status of a Student Application.
+ */
+export enum ApplicationStatus {
+  /**
+   * The application is in draft state and is not Submitted
+   */
+  draft = "draft",

```

Comment:
Please adjust the case to match the ENUM type.

Reply:
how did i miss it